lib/elixir/lib/uri.ex

```diff
@@ -73,42 +73,73 @@ defmodule URI do
   @doc """
   Encodes an enumerable into a query string.
 
+  This function is deprecated. Use `encode_query/2` with `:www_form` as second
+  argument value to achieve the same result.
+  """
+  @doc deprecated: "Use URI.encode_query/2 instead"
+  @spec encode_query(Enum.t()) :: binary
+  def encode_query(enumerable) do
+    encode_query(enumerable, :www_form)
+  end
+
+  @doc """
+  Encodes an enumerable into a query string.
+
   Takes an enumerable that enumerates as a list of two-element
   tuples (for instance, a map or a keyword list) and returns a string
-  in the form of `key1=value1&key2=value2...` where keys and
-  values are URL encoded as per `encode_www_form/1`.
+  in the form of `key1=value1&key2=value2...`.
 
   Keys and values can be any term that implements the `String.Chars`
   protocol with the exception of lists, which are explicitly forbidden.
 
+  You must specify one of the following `encoding` strategies:
+
+    * `:rfc3986` - (recommended) keys and values are encoded following
+      [RFC3986](https://tools.ietf.org/html/rfc3986) specifications.
```

Comment:
Consideration: is `:rfc_3986` a little easier to parse visually? Or is there prior art here with other such identifiers within the stdlib?

Reply:
Good catch! I was under the impression that RFCs were spelled like this, but the page headers on [RFC 3986](https://tools.ietf.org/html/rfc3986) tell me otherwise. Will fix!

Reply:
Should be fixed.